modules/efk-logging-clo-status.adoc

```diff
@@ -0,0 +1,256 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-cluster-status.adoc
+
+[id="efk-logging-clo-status_{context}"]
+= Viewing the status of the Cluster Logging Operator
+
+You can view the status of your Cluster Logging Operator.
+
+.Prerequisites
+
+* Cluster logging and Elasticsearch must be installed.
+
+.Procedure
+
+. Change to the `openshift-logging` project.
++
+----
+$ oc project openshift-logging
+----
+
+. To view the cluster logging status:
+
+.. Get the cluster logging status:
++
+----
+$ oc get clusterlogging instance -o yaml
+----
++
+The output includes information similar to the following:
++
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+
+....
+
+status:  <1>
+  collection:
+    logs:
+      fluentdStatus:
+        daemonSet: fluentd  <2>
+        nodes:
+          fluentd-2rhqp: ip-10-0-169-13.ec2.internal
+          fluentd-6fgjh: ip-10-0-165-244.ec2.internal
+          fluentd-6l2ff: ip-10-0-128-218.ec2.internal
+          fluentd-54nx5: ip-10-0-139-30.ec2.internal
+          fluentd-flpnn: ip-10-0-147-228.ec2.internal
+          fluentd-n2frh: ip-10-0-157-45.ec2.internal
+        pods:
+          failed: []
+          notReady: []
+          ready:
+          - fluentd-2rhqp
+          - fluentd-54nx5
+          - fluentd-6fgjh
+          - fluentd-6l2ff
+          - fluentd-flpnn
+          - fluentd-n2frh
+      rsyslogStatus:
+        Nodes: null
+        daemonSet: ""
+        pods: null
+  curation:  <3>
+    curatorStatus:
+    - cronJobs: curator
+      schedules: 30 3 * * *
+      suspended: false
+  logstore: <4>
+    elasticsearchStatus:
+      shardAllocationEnabled:  all
+      cluster:
+        Active Primary Shards:  5
+        Active Shards:          5
+        Initializing Shards:    0
+        Num Data Nodes:         1
+        Num Nodes:              1
+        Pending Tasks:          0
+        Relocating Shards:      0
+        Status:                 green
+        Unassigned Shards:      0
+      clusterName:             elasticsearch
+      nodeConditions:
+        elasticsearch-cdm-mkkdys93-1:
+      nodeCount:  1
+      Pods:
+        Client:
+          Failed:
+          Not Ready:
+          Ready:
+            elasticsearch-cdm-mkkdys93-1-7f48ffc7c6-mjm7c
+        Data:
+          Failed:
+          Not Ready:
+          Ready:
+            elasticsearch-cdm-mkkdys93-1-7f48ffc7c6-mjm7c
```

Comment:
this is a list, it should have `-` before it like you have in the `visualization` section